ChangeLog.md

```diff
@@ -1,6 +1,15 @@
 # DGtal 0.9
 
 ## New Features / Critical Changes
+- *Geometry Package*
+
+ - New Alpha Thick Segment Computer allowing the recognition of thick digital segment,
```

Comment:
Perhaps you could add that this is also an implementation of the alpha-thick segment of Feschet et (I don't remember who) ?